src/routers/resource_routers/organisation_router.py

```diff
@@ -1,5 +1,25 @@
 from database.model.agent.organisation import Organisation
 from routers.resource_router import ResourceRouter
+from fastapi import UploadFile, File, HTTPException, Query
+from http import HTTPStatus
+from fastapi import APIRouter, Depends
+from sqlmodel import select
+from database.model.agent.organisation import Organisation
+from database.session import get_session
+import base64
+from authentication import KeycloakUser, get_user_or_none, get_user_or_raise
+
+
+ALLOWED_IMAGE_TYPES = {"image/jpeg", "image/png", "image/gif", "image/webp", "image/bmp"}
+MAX_FILE_SIZE = 1 * 1024 * 1024  # 1MB
```

Comment:
```suggestion
MAX_FILE_SIZE_BYTES = 1 * 1024 * 1024  # 1MB
```
Update other places accordingly. Add units of measurements to variables when applicable to avoid confusion.